src/main/java/ru/skypro/homework/dto/comment/Comment.java

```diff
@@ -4,10 +4,10 @@
 
 @Data
 public class Comment {
-    int idAuthor;
-    String authorImage;
-    String firstNameAuthor;
-    int timeCreateComment;
-    int idComment;
-    String description;
+    private int author;
+    private String authorImage;
+    private String authorFirstName;
+    private int createdAt;
```

Comment:
```suggestion
    private long createdAt;
```
int64 - long
int32 = int